frontend/src/components/DropBoxMenu/DropBoxMenu.jsx

```diff
@@ -16,26 +16,28 @@ import styles from './DropBoxStyles.module.scss';
 
 function DropBoxMenu({ menuParams: { title, items } }) {
   const [open, setOpen] = useState(false);
-  const [expanded, setExpanded] = useState(false);
   const anchorRef = useRef(null);
 
-  const handleToggle = () => {
+  const handleToggle = useCallback(() => {
```

Comment:
I don't think we should useCallback for this one I think we can just have:
```suggestion
  const handleToggle = () => setOpen(!open);
```

Reply:
Thanks, you're right!